src/TextInputWithTokens.tsx

```diff
@@ -88,10 +86,7 @@ function TextInputWithTokensInnerComponent<TokenComponentType extends AnyReactCo
     variant: variantProp, // deprecated. use `size` instead
     visibleTokenCount,
     ...rest
-  }: TextInputWithTokensInternalProps<TokenComponentType> & {
-    selectedTokenIndex: number | undefined
-    setSelectedTokenIndex: React.Dispatch<React.SetStateAction<number | undefined>>
-  },
```

Comment:
I don't think `selectedTokenIndex` and `setSelectedTokenIndex` are props that we use in this component

Reply:
You're right. I'm not sure how these types made it here 🤔